### PR TITLE
Port new rendering

### PR DESCRIPTION
**Pull request by Nightenom**
Port rendering fixes from dev branch

Test please, made through github

### CODE REVIEW

File: src/main/java/com/ldtteam/structures/client/BlueprintTessellator.java [hunk 1]
```diff
@@ -86,12 +86,12 @@ private static void preBlueprintDraw()
         GlStateManager.texCoordPointer(LIGHT_TEX_COORD_COMPONENT_SIZE, GL_SHORT, VERTEX_SIZE, LIGHT_TEXT_COORD_COMPONENT_OFFSET);
         OpenGlHelper.setClientActiveTexture(OpenGlHelper.defaultTexUnit);
 
-        GlStateManager.disableCull();
+        // GlStateManager.disableCull();
```
Comment (Asherslab) [inline src/main/java/com/ldtteam/structures/client/BlueprintTessellator.java:89]: code commented out?

File: src/main/java/com/ldtteam/structures/client/BlueprintTessellator.java [hunk 2]
```diff
@@ -86,12 +86,12 @@ private static void preBlueprintDraw()
...
     }
 
     private void postBlueprintDraw()
     {
-        GlStateManager.enableCull();
+        // GlStateManager.enableCull();
```
Comment (Asherslab) [inline src/main/java/com/ldtteam/structures/client/BlueprintTessellator.java:94]: same here

File: src/main/java/com/ldtteam/structures/client/BlueprintTessellator.java (hunk 1 above)
Comment (Nightenom) [reply src/main/java/com/ldtteam/structures/client/BlueprintTessellator.java:89]: No need to disable culling
Left here for anyone to resurrect this

Comment (Asherslab) [reply src/main/java/com/ldtteam/structures/client/BlueprintTessellator.java:89]: okay, just making sure you didn't accidentally leave it in from testing something

**Pull request merged**